keras_nlp/models/bert/bert_backbone_test.py

```diff
@@ -25,74 +25,62 @@
 
 class BertBackboneTest(tf.test.TestCase, parameterized.TestCase):
     def setUp(self):
-        self.model = BertBackbone(
-            vocabulary_size=1000,
-            num_layers=2,
-            num_heads=2,
-            hidden_dim=64,
-            intermediate_dim=128,
-            max_sequence_length=128,
+        self.backbone = BertBackbone(
+            vocabulary_size=10,
+            num_layers=1,
```

Comment:
I'd actually recommend picking 2 (for layers/heads) for at least one test since "1" tends to be a boundary and may not get full coverage

Reply:
sgtm!